scripts/ingest_events_bulk.py

```
@@ -0,0 +1,242 @@
+                self.stats["duplicates"] += 1
+    
+    def print_stats(self) -> None:
+        """Print ingestion statistics."""
+        duration = self.stats["end_time"] - self.stats["start_time"]
+        throughput = self.stats["total"] / duration if duration > 0 else 0
+        
+        print("\n" + "=" * 70, file=sys.stderr)
+        print("📊 INGESTION STATISTICS", file=sys.stderr)
+        print("=" * 70, file=sys.stderr)
+        
+        print(f"\n⏱️  Performance:", file=sys.stderr)
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)